spark-testing/spark_utils.py

```diff
@@ -26,7 +26,7 @@
 SPARK_SERVICE_ACCOUNT = os.getenv("SPARK_SERVICE_ACCOUNT", "spark-service-acct")
 SPARK_SERVICE_ACCOUNT_SECRET = os.getenv("SPARK_SERVICE_ACCOUNT_SECRET", "spark-service-acct-secret")
 SPARK_SERVICE_NAME = os.getenv("SPARK_SERVICE_NAME", "spark")
-FOLDERED_SPARK_SERVICE_NAME = "/path/to/" + SPARK_SERVICE_NAME
+FOLDERED_SPARK_SERVICE_NAME = "path/to/" + SPARK_SERVICE_NAME
```

Comment:
In `dcos-commons` we explicitly have a leading `/`. There may be another reason why the tooling is causing this to fail.